cookbooks/fb_iptables/files/default/fb_iptables_reload.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC2086,SC2018,SC2019,SC1090,SC2064,SC2124,SC2181,SC2002,SC2153,SC2046,SC2173
```

Comment:
FYI - this is because this thing fails shellcheck badly and so I had to either lint it, or disable all this to merge it internally. Then I figured you wuoldn't want all of that change in this PR, so I'm first submitting the fixes... then I will submit fugure PRs to lint this.

Reply:
Yeah that's fine. Add a TODO or a comment so we don't forget though.

Reply:
Also, add SC2230 to the list, it's failing that too internally

Reply:
That IS the TODO :) Also, it has to be on the 2nd line, so I can add another line below it saying fix, but I think the disabling all of lint is an obvious TODO. I also have an internal task to fix it
Yeah I can add that rule... let me know how testing goes in case I need to make any other changes.

Reply:
Looking good so far, I'll merge it once you update this.